banzai_nres/utils/wavelength_utils.py

```diff
@@ -20,8 +20,8 @@ def identify_features(data, err, mask=None, nsigma=2., fwhm=6.0, **kwargs):
              catalog of features (e.g. emission lines). features['xcentroid'][j] gives the horizontal pixel position
              of the jth feature. features['ycentroid'][j] gives the y pixel (vertical) position of the jth feature.
     """
-    daofind = DAOStarFinder(fwhm=fwhm, threshold=np.nanmin(err) * nsigma, exclude_border=True, **kwargs)
-    features = daofind(data, mask=mask)
+    daofind = DAOStarFinder(fwhm=fwhm, threshold=nsigma, exclude_border=True, **kwargs)
+    features = daofind(data/err, mask=mask)
```

Comment:
PEP8

Reply:
will change data/err = data / err .

Let me know if there is another PEP8 violation